database_output_twitterのデータモデリング

imahoritatsuki opened the pull request:
## 課題のリンク

*https://github.com/happiness-chain/practice/blob/main/database/design.md

## やったこと

* twitter.erファイルにER図のコードを記述
* twitter.erファイルをPDFで出力


## 動作確認方法

* twitter.erファイルのコードをご確認くださ
* PDFで出力したER図をご確認ください。
[out.pdf](https://github.com/imahoritatsuki/hc_practice/files/12775060/out.pdf)
い。


## その他

* お忙しいとこと恐縮ですが、よろしくお願いいたします。

Commit:
- twitter.erファイルにER図のコードを記述
- twitter.erをPDFに出力

yamaryo416 commented on twitter_erd/twitter.er:
```diff
@@ -0,0 +1,107 @@
+
+[retweet]
+  *id {label: "CHAR, not null"}
+  original_post_id {label: "CHAR, not null, FK(posts)"}
```
こちらもpost_idでいいかなと思いました。

yamaryo416 commented on twitter_erd/twitter.er:
```diff
@@ -0,0 +1,107 @@
+
+[follows]
+  *id {label: "CHAR, not null"}
+  user_id {label: "CHAR, not null, FK(users)"}
```
follower_idとfollowed_idがいいかなと思いました。

https://zenn.dev/odentravel/books/e69a157daeecb3/viewer/086fe9

yamaryo416 reviewed (approved):
LGTM!

Commit:
レビューの対応
- retweetエンティティのoriginal_post_idをpost_idに変更
- followsエンティティのidをfollower_idに変更
- followsエンティティのuser_idをfollowed_idに変更
